_posts/2017-03-06-internetfreedomfestival.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "Internet Freedom Festival 2017"
+date:   2017-03-06
+categories: design internet-freedom activism
+eventDate: Tue 6 - Fri 10 March 2017
+location: Calle de Juan Verdeguer 16, 46024 – Valencia, Spain
+time: All day
+status: upcoming
+---
+
+[The Internet Freedom Festival](https://internetfreedomfestival.org/) creates a common space where diverse communities working against and affected by censorship and surveillance can come together to teach, plan and act.
+
+The [Communications and Design theme](https://internetfreedomfestival.org/#themes) will connect designers, users, and developers to keep the tools secure and usable.
+
+This conjunction will lead to activities such as UX/UI brainstorms for new and improved tools, usability workshops for developers, and opportunities for users to share their highly valuable experience and recommendations.
+
+Sessions can take the format of: workshops, trainings, hackathons, presentations, panel discussions, networking events, or if you have a different format, suggest it!
+
+The call for session proposals is for designers, user researchers (and everyone working in user experience) who want to contribute their knowledge, experience or time to projects working on Internet freedom, anti-censorship and surveillance.
+
+[Closing date for submitting a session proposal is the 2nd of October] (https://internetfreedomfestival.org/host-a-session/)
```

Comment:
the submission was for 02.10.2016? so this already passed. Maybe we are keeping it just as reference?